okhttp/src/main/kotlin/com/connectrpc/okhttp/ConnectOkHttpClient.kt

```diff
@@ -43,24 +43,38 @@ import okio.BufferedSink
 import java.io.IOException
 import java.io.InterruptedIOException
 import java.net.SocketTimeoutException
+import java.time.Duration
 
 /**
  * The OkHttp implementation of HTTPClientInterface.
  */
 class ConnectOkHttpClient @JvmOverloads constructor(
-    unaryClient: OkHttpClient = OkHttpClient(),
-    streamClient: OkHttpClient = unaryClient,
+    private val unaryClient: OkHttpClient = OkHttpClient(),
+    // TODO: remove this; a separate client is only useful for configuring separate timeouts,
```

Comment:
This is not just for timeouts IIRC - you may also want to configure pings on clients for long lived streams but not on ones for unary calls.

Reply:
Here's some background on the two clients (aside from timeouts which will be much better behaved with this PR): https://github.com/connectrpc/connect-kotlin/issues/13

Reply:
It's unclear why you'd want PINGs on one client and not the other. If you have created a connection that uses PINGs, just use that same client for both unary and stream traffic. I don't see any downside to that. It wouldn't increase the PING activity since you're not creating two clients that both use PING -- but instead using the PINGing client for all RPCs.

As far as the read timeout being different between unary and streams, that is now even more configurable (based on `MethodSpec`) via the timeout oracle. So you can set the read timeout to zero (or use `ConnectOkHttpClient.configureClient`) and it should be appropriate for both unary and stream traffic.

@kohenkatz, please take a look at this thread and PR and let me know what you think. Note that this PR is not removing the second client -- not yet. But I don't think it will be needed anymore and would appreciate your input, like if you see other reasons to keep the second client in future versions.